src/Toolkit/Toolkit/UI/Controls/Compass/Compass.cs

```diff
@@ -44,10 +45,25 @@ public partial class Compass : Control
         public Compass()
             : base()
         {
+            ResetOnClick = true;
             Initialize();
         }
 #endif
 
+        /// <summary>
+        /// Gets called, when the Compass is clicked
+        /// </summary>
+        public event EventHandler CompassClicked;
```

Comment:
Should it just be `Clicked` ?
Also should it be a cancellable RoutedEvent (and if cancelled won't trigger reset)